src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -63,12 +65,30 @@ private static void setupDrivers(Application application) {
         DriverFeature.addDriver("Precise logger driver", preciousLoggerDriver);
 
         DrawPanelController drawerController = DrawerFeature.getDrawerController();
-        Job2dDriver driver = new LineDriverAdapter(drawerController, LineFactory.getBasicLine(), "basic");
-        DriverFeature.addDriver("Line Simulator", driver);
-        DriverFeature.getDriverManager().setCurrentDriver(driver);
 
-        driver = new LineDriverAdapter(drawerController, LineFactory.getSpecialLine(), "special");
-        DriverFeature.addDriver("Special line Simulator", driver);
+        // TransformationDriver for Line Simulator
+        TransformingDriver commonTransformedDriver = new TransformingDriver(new LineDriverAdapter(drawerController, LineFactory.getBasicLine(), "basic"));
+        commonTransformedDriver.addModifier(new ScalingModifier(1.0, 1.0));
+        commonTransformedDriver.addModifier(new RotationModifier(0));
+        commonTransformedDriver.addModifier(new XAxisMovingModifier(0));
+        commonTransformedDriver.addModifier(new YAxisMovingModifier(0));
```

Comment:
What is the point of those modifiers? They do not change anything. 
Propose a transformation (describe it shortly in a driver label)